### PR TITLE
[External Contribution] Added scripts to package up the sample apps for easier chat hero migration consumption

### PR DESCRIPTION
**Pull request by alex-kwan**
# What
Added scripts to package up the sample apps for easier chat hero migration consumption

# Why
There are several manual steps required to get the sample apps into the shape we need to copy and paste over the existing hero samples. Added in some basic copy/rename/move kind of scripts to make this eaiser.

# How Tested
Ran this locally and validated the sample-dist directory where the files are moved to

### CODE REVIEW

File: samples/CallWithChat/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "test:stable": "echo skip",
     "preprocess": "cpx \"./src/**\" ./preprocessed && babel ./src --out-dir ../preprocessed --extensions \".ts,.tsx\" --keep-file-extension --config-file ./.babelrc.js --relative",
     "preprocess:stable-flavor": "rushx _stable-flavor \"rushx preprocess && rushx lint:fix && rushx prettier\"",
+    "port:sample": "rushx preprocess:stable-flavor && cpx \"./preprocessed/**\" sample-dist/src && cpx \"./Media/**\" sample-dist/Media && cpx \"./public/**\" sample-dist/public",
```
Comment (emlynmac) [inline samples/CallWithChat/package.json:20]: Where is the documentation around how this new command should be used?

**Review by emlynmac — APPROVED** [review]
We probably need some steps in the doc [here](https://github.com/Azure/communication-ui-library/blob/main/docs/releases/hero-samples.md) as to how to use this command?

File: samples/Chat/package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "test:ci-coverage": "echo skip",
     "preprocess": "cpx \"./src/**\" ./preprocessed && babel ./src --out-dir ../preprocessed --extensions \".ts,.tsx\" --keep-file-extension --config-file ./.babelrc.js --relative",
     "preprocess:stable-flavor": "rushx _stable-flavor \"rushx preprocess && rushx lint:fix && rushx prettier\"",
+    "port:sample": "rushx preprocess:stable-flavor && cpx \"./preprocessed/**\" sample-dist/src && cpx \"./Media/**\" sample-dist/Media && cpx \"./public/**\" sample-dist/public",
```
Comment (PorterNan) [inline samples/Chat/package.json:18]: Not related to this change, but when I am reading _stable-flavor file it is like:

"_by-flavor": "rushx _current-flavor && env-cmd -f ../../common/config/env/.env --use-shell",

It shows you current flavor then run the command under stable flavor - which might be confusing, I think we can remove "rushx _current-flavor &&" part of it, but doesn't have to be in this PR

**Review by PorterNan — APPROVED** [review]

File: samples/Chat/package.json (hunk 2 above)
Comment (alkwa-msft) [reply samples/Chat/package.json:18]: @PorterNan  can we log this as a bug that we can close quite quickly? I dont want to lose track of this suggestion.

**Pull request closed without merging**